src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -107,7 +108,7 @@ class AttitudeEstimatorQ
 	bool		_task_should_exit = false;	/**< if true, task should exit */
 	int		_control_task = -1;		/**< task handle for task */
 
-	int		_sensors_sub = -1;
+	uORB::SubscriptionBlocking<sensor_combined_s>	_sensors_sub{ORB_ID(sensor_combined)};
```

Comment:
is this just a value set or is there an alloc? What happens here on failure?

Reply:
It's a pointer + metadata to the underlying publication and there's no allocation.